search/README.md

```diff
@@ -222,6 +222,13 @@ mapping@org.sakaiproject.search.elasticsearch.ElasticSearchIndexBuilder=
                 "index": "not_analyzed",                               \
                 "store": "yes"                                         \
             },                                                         \
+            "creatordisplayname": {                                    \
```

Comment:
Was thinking about this and would add
creatorName
creatorId

Reply:
Ok, I'll update it.

Reply:
@ern I've updated it to index the display name and user name (eid). Nobody wants to search on a UUID, so I left the user id out.

Reply:
And, I've now added the creator's user id. Maybe it will be useful.